services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/thread/PortBlinker.groovy

```diff
@@ -0,0 +1,49 @@
+package org.openkilda.functionaltests.helpers.thread
+
+import org.openkilda.testing.model.topology.TopologyDefinition.Switch
+import org.openkilda.testing.service.northbound.NorthboundService
+
+class PortBlinker extends Thread {
+    private NorthboundService northbound
+    private volatile boolean running
+
+    final Switch sw
+    final int port
+    Date timeStarted
+    Date timeStopped
+
+    PortBlinker(NorthboundService northbound, Switch sw, int port) {
```

Comment:
Controllable delay between port up/down events would be nice.

Reply:
Agree. I thought about this after push, but it is currently not needed anywhere. The class can be extended with new functionality pretty easy, so this can be added on-demand later